.github/workflows/markdowns.yml

```diff
@@ -32,13 +30,18 @@ jobs:
           translationsPath: 'CompassOfWomanhood/translations'
 
       - name: Commit & Push changes
-        uses: actions-js/push@master
+        uses: actions-js/push@v1.3
         with:
           author_email: publisher@udiknedormin.github.io
           author_name: Github Publisher [bot]
           github_token: ${{ secrets.GITHUB_TOKEN }}
           message: Publishing merged markdowns for branch ${{ github.head_ref }}
```

Comment:
Notes from the debug session:

Based on this message from two commits, on-pr-create and on-push, it seemed like `github.head_ref` was empty. A dump confirmed that. Fortunately, there is a Github Action to add `head_ref` based on `ref`, which we discussed earlier today.